taxi/urls.py

```diff
@@ -19,9 +19,8 @@
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
-    path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
-    ),
+    path("drivers/<int:pk>/", DriverDetailView.as_view(),
+         name="driver-detail"),
```

Comment:
The URL pattern for `driver-detail` is correctly defined. Ensure that the `DriverDetailView` view is implemented and handles the `pk` parameter appropriately.